src/List/ListItem.styles.ts

```diff
@@ -2,6 +2,7 @@ import styled, { css } from 'styled-components';
 
 export interface ListItemProps extends React.HTMLAttributes<HTMLDivElement> {
   disabled?: boolean;
+  platform?: boolean;
```

Comment:
If the list component is setup with context the porps shouldn't need to be passed on each item.

Reply:
As discussed, I'll rewrite it to use context after release